album/QMEP-0002.toml

```
@@ -0,0 +1,46 @@
+[album]
+title = "南北极星 Vol.1"
+artist = "洛天依、乐正绫、星尘"
+date = 2016-11-30
+type = "normal"
+catalog = "QMEP-0002"
+tags = ["洛天依", "乐正绫", "星尘", "平行四界 EP"]
```

```suggestion
tags = ["洛天依", "乐正绫", "星尘", "平行四界", "EP"]
```
May be better to add a single independent tag named `EP`, and also `album` and so on.

---

Making `EP`/`Single`/`Album` a single category might be strange. Something like `平行四界:default` might be better for those albums(without series).

---

Or `平行四界:EP`.

---

`平行四界:EP` Seems good.